src/user-agent-util.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const os = require('os');
+const packagejson = require('../package.json');
+
+module.exports = `${packagejson.name} (Project Helix) ${packagejson.version} - ${os.platform()} ${os.release} +${packagejson.homepage}`;
```

Comment:
why an url in the user agent string?

Reply:
It seems to be a good practice to put the URL into the UA String for bots: https://www.keycdn.com/blog/web-crawlers/

In addition, GitHub requests to put in the application name, so that they can contact us in case of issues. I thought a URL would be pretty unambiguous.

Reply:
ok. +1